content/en/continuous_integration/tests/junit_upload.md

```diff
@@ -322,9 +329,109 @@ For mobile apps (Swift, Android):
+
+`/testcase//property[@name='property-name']`
+: The `value` attribute from the `<property name="property-name" value="value">` inside the current `<testcase>`.
+
+Examples:
+
+{{< tabs >}}
+
+{{% tab "Test suite from @classname" %}}
+By default the `test.suite` tag of the tests is read from `<testsuite name="suite name">`. However some plugins might report a better value in `<testcase classname="TestSuite">`.
+
+To change `test.suite` tags from `value 1`, `value 2` to `SomeTestSuiteClass`, `OtherTestSuiteClass`:
+
+{{< code-block lang="xml" >}}
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite tests="1" failures="0" time="0.030000" name="value 1">
+    <testcase classname="SomeTestSuiteClass" name="test_something" time="0.030000"></testcase>
+  </testsuite>
+  <testsuite tests="1" failures="0" time="0.021300" name="value 2">
+    <testcase classname="OtherTestSuiteClass" name="test_something" time="0.021300"></testcase>
+  </testsuite>
+</testsuites>
+{{< /code-block >}}
+
+{{< code-block lang="bash" >}}
+datadog-ci junit upload --service service_name \
+  --xpath-tag test.suite=/testcase/@aclassname ./junit.xml
+{{< /code-block >}}
+
+{{% /tab %}}
+
+{{% tab "Tag from attribute" %}}
+To add `custom_tag` to each test with values `value 1`, `value 2`.
+
+{{< code-block lang="xml" >}}
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite tests="1" failures="0" time="0.020000" name="SomeTestSuiteClass">
+    <testcase name="test_something" time="0.010000" attr="value 1"></testcase>
+    <testcase name="test_other" time="0.010000" attr="value 2"></testcase>
+  </testsuite>
+</testsuites>
+{{< /code-block >}}
+
+{{< code-block lang="bash" >}}
+datadog-ci junit upload --service service_name \
+  --xpath-tag custom_tag=/testcase/@attr ./junit.xml
+{{< /code-block >}}
+
+{{% /tab %}}
+
+{{% tab "Tag from testsuite property" %}}
+To add `custom_tag` to each test with values `value 1`, `value 2`.
+
+{{< code-block lang="xml" >}}
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite tests="1" failures="0" time="0.030000" name="SomeTestSuiteClass">
+    <properties>
+      <property name="prop" value="value 1"></property>
+    </properties>
+    <testcase name="test_something" time="0.030000" attr="value 1"></testcase>
+  </testsuite>
+  <testsuite tests="1" failures="0" time="0.021300" name="OtherTestSuiteClass">
+    <properties>
+      <property name="prop" value="value 1"></property>
+    </properties>
+    <testcase name="test_something" time="0.021300" attr="value 1"></testcase>
+  </testsuite>
+</testsuites>
+{{< /code-block >}}
+
+{{< code-block lang="bash" >}}
+datadog-ci junit upload --service service_name \
+  --xpath-tag custom_tag=/testcase/..//property[@name=\'prop\'] ./junit.xml
+{{< /code-block >}}
+
+**Note:** the escaping of quotes with `\'`. The quotes are required and bash removes them if not escaped.
```

Comment:
Not sure it's the best way to put it, but the current is a bit difficult to understand.
```suggestion
**Note:** the quotes are required. Also, you have to escape quotes with slashes, e.g. `\'`.
```